src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,17 @@
-export const UserInfo = () => {};
+// import usersFromServer from '../../api/users';
+
+import { User } from '../types/User';
+
+interface Props {
+  user: User;
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  // const user = usersFromServer.find(user => user.id === userId);
+
+  return (
+    <a className="UserInfo" href={`mailto:${user?.email}`}>
+      {user?.name}
+    </a>
```

Comment:
```suggestion
    <a className="UserInfo" href={`mailto:${user.email}`}>
      {user.name}
    </a>
```
think, it should work without "?"